packages/auth/src/signature/middleware.test.ts

```diff
@@ -402,7 +402,7 @@ describe('Signature Service', (): void => {
         grantContinueHttpsigMiddleware(ctx, next)
       ).rejects.toHaveProperty('status', 401)
 
-      scope.isDone()
+      // scope.done() // TODO: fails
```

Comment:
This should work when test above is fixed.
```suggestion
      scope.done()
```